ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh

```diff
@@ -18,7 +18,10 @@ GSHEET_KEY_LOCATION="/ga-gsheet/gcp-sa-account"
 export GSHEET_KEY_LOCATION
 
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
-pushd e2e-benchmarking/workloads/kube-burner
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+#export EXTRA_FLAGS="--pods-per-node=$PODS_PER_NODE "
+#export NAMESPACED_ITERATIONS=${NAMESPACED_ITERATIONS:-false}
+#export EXTRA_FLAGS=${EXTRA_FLAGS:-}
 export WORKLOAD=node-density-heavy
 
 export CLEANUP_WHEN_FINISH=true
```

Comment:
We can capture these configurable variables in [openshift-qe-node-density-heavy-ref.yaml](https://github.com/openshift/release/pull/41745/files#diff-fd8637f16f5b639fef15ec39022b495a5a84001e7dbf87407f22cceb3dbbbdc5) and override
as required in [openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml](https://github.com/openshift/release/pull/41745/files#diff-99f37dbb0b40d6b2cafd605b48b7b13ab9194dedfc1b6986543d7a669cbf0e54)

Reply:
Right now, we have these for all workloads and I agree that they have to be moved to upper level. 
I think they should be tackled as part of https://issues.redhat.com/browse/OCPQE-15384